deltachat-ffi/src/lib.rs

```diff
@@ -4424,3 +4425,80 @@ pub unsafe extern "C" fn dc_accounts_get_next_event(
         .map(|ev| Box::into_raw(Box::new(ev)))
         .unwrap_or_else(ptr::null_mut)
 }
+
+#[cfg(feature = "jsonrpc")]
```

Comment:
instead of having all these cfgs, please put the whole thing into its own `mod` and cfg it

Reply:
wasn't sure that would work, thanks for the hint! 😄